switch/switch_main.ml

```diff
@@ -171,13 +171,9 @@ let make_server config =
       on_disk_queues := qs;
       info "Reading the redo-log from %s" redo_log_path;
       Block.connect ~buffered:true redo_log_path
-      >>= (function
-          | `Ok block -> Lwt.return block
-          | `Error `Unknown s -> Lwt.fail (Failure (Printf.sprintf "Unexpected failure when opening block: %s" s))
-          | `Error _ -> Lwt.fail (Failure "Other error"))
       >>= fun block ->
       let open Lwt_result in
-      Redo_log.start ~flush_interval:5. block (process_redo_log statedir)
+      Redo_log.start ~flush_interval:5_000_000_000L block (process_redo_log statedir)
```

Comment:
This is because shared-block-ring broke the interface and moved from seconds in float to nanoseconds in int64 (see https://github.com/mirage/shared-block-ring/commit/cdab0b352b92719b50f441367e618c3377c6631c )